### DISCUSSION TITLE
.all

### DISCUSSION THREAD

**Discussion by ShoveD01** [bangcorrupt/hacktribe#116] [General]
Has anyone got an e2sSample.all file that has the factory samples moved from 19 onwards to 501 onwards.

**Comment by BKLronin** [comment]
Nice try :D

**Comment by maks** [comment]
Why are people so concerned about getting the factory samples? I've only ever seen comments online complaining that the factory samples actually aren't very good at all.

**Reply by BKLronin** [reply]
I just heard the ones from the esx. Kind of ok but there are so much free samples out there usually not aproblem to get a nice set together.

I kinda miss the synth drums a little. Just because I got used to them over so many years.

**Reply by maks** [reply]
Yes it would be nice to be able to use the drums of my e2 synth. The way I understand it, they are synthesized and not samples (or maybe I'm wrong?) so I guess it maybe be possible to still access them with future versions of hacktribe?

**Reply by bangcorrupt** [reply]
They're samples, the flash format is the same as the sampler.  The Edit knob applies one of six flavours of drive, from a subtle top boost to all out destruction.  I'm hoping I can add it all to hacktribe at some point, along with key/velocity mapping.

**Reply by maks** [reply]
Cool thanks, in that case I'm really not worried about losing them, I already have several free or cheap sample packs I've already downloaded to try out and use and I'm having great time at the moment trying out carving beats! 😀

**Reply by BKLronin** [reply]
Me neither they were always hard to get into a real sweet spot. There are also not much different from most classic drum samples I heard over the years.

The drum samples in the carved beats set really ate a lot of time as I don't synthesize drums that often. I definitely made two real nice tracks out of them(not on electrode). (Electro and nice hip-hop)

**Reply by chaocrator** [reply]
factory samples have some really good & usable stuff — that's why. 
i myself did not want to reinvent what worked well for me (certain drum/percussion stuff i'm used to).

**Reply by BKLronin** [reply]
Its a little different for me. I searched for this to get rid of the factory samples. So I don´t mind at all.

**Comment by ShoveD01** [comment]
I’ve actually got them, but they are at 19 onwards . They can’t be moved with the editor.
So I was only asking  if anyone had moved them already, so as to save me going back to the old hack , exporting and dragging back into the editor to 501 onwards.

**Reply by bangcorrupt** [reply]
[FAQ](https://github.com/bangcorrupt/hacktribe/wiki/faq#i-cant-import-my-samples)

**Comment by ShoveD01** [comment]
Thanks , I was and always have used the other editor, where you can’t move them from one area to the other. ( unless I’m missing something) . Never been able to get oe2sle working.
I’ll have another go.

**Reply by BKLronin** [reply]
You can afair in the python one. I think we can make an all in one containing executable version from it. I can remember havin problems on windows as well.

**Comment by ecco00** [comment]
@ShoveD01 i could send you an email regarding this topic if you're still interested.

**Reply by ShoveD01** [reply]
Thanks , I've sorted it.
